cmd/wsh/cmd/wshcmd-file-util.go

```
@@ -210,3 +209,25 @@ func streamFileList(zoneId string, path string, recursive bool, filesOnly bool)
 
 	return resultChan, nil
 }
+
+func fixRelativePaths(path string) (string, error) {
+	conn, err := connparse.ParseURI(path)
+	if err != nil {
+		return "", err
+	}
+	if conn.Scheme == connparse.ConnectionTypeWsh {
+		fixedPath := conn.Path
```

golangci-lint (1.62.2)

ineffectual assignment to fixedPath

(ineffassign)